src/common/format.ts

```diff
@@ -0,0 +1,90 @@
+import { join as joinPath } from 'path';
+
+import { CompleteSuperfaceTestConfig } from '..';
+
+const ISO_DATE_REGEX =
+  /(\d{4})-(\d{2})-(\d{2})T(\d{2}):(\d{2}):(\d{2}(?:\.\d*)?)((-(\d{2}):(\d{2})|Z)?)/gm;
+
+export const removeTimestamp = (payload: string): string =>
+  payload.replace(ISO_DATE_REGEX, '');
+
+export function getFixtureName(sfConfig: CompleteSuperfaceTestConfig): string {
+  const { profile, provider, useCase } = sfConfig;
+
+  return joinPath(
+    profile.configuration.id,
+    provider?.configuration.name,
+    useCase?.name
+  );
+}
+
+/**
+ * SUPERFACE_LIVE_API="*"
+ * SUPERFACE_LIVE_API="profile*"
+ * SUPERFACE_LIVE_API="scope/profile:provider:usecase*"
+ */
+export function matchWildCard(
+  sfConfig: CompleteSuperfaceTestConfig,
+  superfaceEnv?: string
+): boolean {
+  if (superfaceEnv === undefined) {
+    return false;
+  }
+
+  const { profile, provider, useCase } = sfConfig;
+  const [profilePayload, providerPayload, usecasePayload] =
+    superfaceEnv.split(':');
+
+  // parse profile
+  if (profilePayload) {
+    if (profilePayload.endsWith('*')) {
+      const regex = RegExp(
+        `^${profilePayload.substr(0, profilePayload.length - 1)}.`
```

Comment:
Seems like it doesn't need to be there, but originally I wanted to match any other character after payload, so correctly it would be `.*` at the end.